cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryFeatureDataset.java

```diff
@@ -0,0 +1,240 @@
+        NetcdfDataset ds = ucar.nc2.dataset.NetcdfDataset.acquireDataset(null, DatasetUrl.findDatasetUrl(location), enhanceMode, -1, null, null);
+        return new SimpleGeometryFeatureDataset(ds);
+    }
+
+    /**
+     * Create a SimpleGeometryFeatureDataset from a NetcdfDataset.
+     *
+     * @param ncd underlying NetcdfDataset, will do Enhance.CoordSystems if not already done.
+     * @throws java.io.IOException on read error
+     */
+    public SimpleGeometryFeatureDataset(NetcdfDataset ncd) throws IOException {
+        this.ncd = ncd;
+        Set<NetcdfDataset.Enhance> enhance = ncd.getEnhanceMode();
+        if(enhance == null || enhance.isEmpty()) enhance = NetcdfDataset.getDefaultEnhanceMode();
+        ncd.enhance(enhance);
+    }
+
+    // stuff to satisfy ucar.nc2.dt.TypedDataset
+    public String getTitle() {
+        String title = ncd.getTitle();
+        if (title == null)
+            title = ncd.findAttValueIgnoreCase(null, CDM.TITLE, null);
+        if (title == null)
+            title = getName();
+        return title;
+    }
+
+    public String getDescription() {
+        String desc = ncd.findAttValueIgnoreCase(null, "description", null);
+        if (desc == null)
+            desc = ncd.findAttValueIgnoreCase(null, CDM.HISTORY, null);
+        return (desc == null) ? getName() : desc;
+    }
+
+    public String getLocation() {
+        return (ncd != null) ?  ncd.getLocation() : "";
+    }
+
+    @Override
+    public CalendarDateRange getCalendarDateRange() {
+        return null;
+    }
+
+    @Override
+    public CalendarDate getCalendarDateStart() {
+        return null;
+    }
+
+    @Override
+    public CalendarDate getCalendarDateEnd() {
+        return null;
+    }
+
+    @Override
+    public LatLonRect getBoundingBox() {
+        return null;
+    }
+
+    public List<Attribute> getGlobalAttributes() {
+        return ncd.getGlobalAttributes();
+    }
+
+    public Attribute findGlobalAttributeIgnoreCase(String name) {
+        return ncd.findGlobalAttributeIgnoreCase(name);
+    }
+
+    @Override
+    public List<VariableSimpleIF> getDataVariables() {
+        return null;
+    }
+
+
+    public VariableSimpleIF getDataVariable(String shortName) {
+        return ncd.getRootGroup().findVariable(shortName);
+    }
+
+    public NetcdfFile getNetcdfFile() {
+        return ncd;
+    }
+
+    @Override
+    public void close() throws IOException {
+        ncd.close();
+    }
+
+    @Override
+    public long getLastModified() {
+        return 0;
+    }
+
+    @Override
+    public void setFileCache(FileCacheIF fileCache) {
+        ncd.setFileCache(fileCache);
+    }
+
+    @Override
+    public void getDetailInfo(Formatter sf) {
+        ncd.getDetailInfo(sf);
+    }
+
+    /**
+     * the name of the dataset is the last part of the location
+     * @return the name of the dataset
+     */
+    public String getName() {

```

Comment:
Is `location` guaranteed to be a URL, URI or path? If so, would it be relevant to have this method delegate to utility methods in `URL`, `URI`, `Paths` or `Path`?

Reply:
leaving implementation same as in GridDataset based on our conversation